### PR TITLE
chore: update VHD version to 04.30.2019

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
**Reason for Change**:

Includes pre-pulled components for Azure Stack.

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by CecileRobertMichon** [comment]
/hold

image is not yet published

**Comment by CecileRobertMichon** [comment]
@jackfrancis are there any CIS E2E tests that can be "un-skipped" when this one lands?

**Comment by jackfrancis** [comment]
The cloud-init and pw validation tests are still gated because a few changes went in after the last VHD was published, so those are able to be re-enabled for VHD scenarios

File: test/e2e/kubernetes/kubernetes_test.go [hunk 1]
```diff
@@ -231,32 +231,28 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 		})
 
 		It("should validate all cloud-init-paved files", func() {
-			if eng.ExpandedDefinition.Properties.IsUbuntuDistroForAllNodes() {
```
Comment (CecileRobertMichon) [inline test/e2e/kubernetes/kubernetes_test.go]: if this diff is hard to read try using https://github.com/Azure/aks-engine/pull/1184/files?w=1

**Comment by jackfrancis** [comment]
This is failing because these files under `/var/log` have "everyone" read access:

```
$ sudo find /var/log -type f -perm '/o+r'
/var/log/wtmp
/var/log/cloud-init.log
/var/log/apt/history.log
/var/log/azure/custom-script/handler.log
/var/log/azure/docker-monitor.timer-status.log
/var/log/azure/etcd-status.log
/var/log/azure/kubelet-status.log
/var/log/azure/rpcbind-status.log
/var/log/azure/cluster-provision.log
/var/log/azure/docker-status.log
/var/log/azure/systemd-journald-status.log
/var/log/azure/rpc-statd-status.log
/var/log/kubeaudit/audit.log
/var/log/unattended-upgrades/unattended-upgrades-shutdown.log
```

File: test/e2e/kubernetes/kubernetes_test.go [hunk 2]
```diff
@@ -345,32 +345,28 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 		})
 
 		It("should validate password enforcement configuration", func() {
-			if eng.ExpandedDefinition.Properties.IsUbuntuDistroForAllNodes() {
```
Comment (CecileRobertMichon) [inline test/e2e/kubernetes/kubernetes_test.go]: if this diff is hard to read try using https://github.com/Azure/aks-engine/pull/1184/files?w=1

**Comment by CecileRobertMichon** [comment]
@jackfrancis I un-did the change to run the cloud-init files E2E so we can unblock the Azure Stack VHD stuff while we figure out what's wrong with the way we're setting file permissions and publish a new VHD if needed.

**Comment by jackfrancis** [comment]
@CecileRobertMichon FYI I pushed up a commit to fix failing Windows E2E

**Comment by jackfrancis** [comment]
@CecileRobertMichon FYI I pushed up a commit to fix failing Windows E2E

**Pull request merged**